src/style.css

```diff
@@ -1 +1,52 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.box {
+  background-color: #fff;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__list {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
You can make it shorter, use just - https://developer.mozilla.org/ru/docs/Web/CSS/background